pkgs/applications/networking/instant-messengers/freetalk/default.nix

```diff
@@ -13,6 +13,16 @@ stdenv.mkDerivation rec {
     sha256 = "09jwk2i8qd8c7wrn9xbqcwm32720dwxis22kf3jpbg8mn6w6i757";
   };
 
+  patches = [
+    # Pull pending patch for -fno-common tuulchain support:
+    #   https://github.com/GNUFreetalk/freetalk/pull/39
+    (fetchpatch {
+      name = "fno-common.patch";
+      url = "https://github.com/GNUFreetalk/freetalk/commit/bf8c326da21bb3c6bdc015c97ca26cea0e433065.patch";
```

Comment:
Looks like upstream did a squash-and-merge, so maybe it would be better to use the mainline commit https://github.com/GNUFreetalk/freetalk/commit/f04d6bc8422be44cdf51b29c9a4310f20a18775a? LGTM otherwise

Reply:
Good point! Changed commit ID.